assets/chat/js/chat.js

```diff
@@ -415,7 +415,13 @@ class Chat {
 
     // ESC
     document.addEventListener('keydown', (e) => {
-      if (isKeyCode(e, KEYCODES.ESC)) ChatMenu.closeMenus(this); // ESC key
+      if (isKeyCode(e, KEYCODES.ESC)) {
+        const activeView = this.getActiveWindow().scrollplugin;
```

Comment:
You can go ahead and add a method to `ChatWindow` that scrolls it to the bottom so you don't have to interface with its scroll plugin directly.

Reply:
Good idea! Added methods on ChatWindow to interface with its scroll plugin